src/lib/y2storage/dialogs/proposal.rb

```diff
@@ -310,6 +311,22 @@ def list(items)
         Yast::HTML.List(items)
       end
 
+      def encryption_authentication
+        return "" unless @devicegraph.encryptions&.length
+
+        auth_list = []
+        @devicegraph.encryptions&.each do |d|
+          if d.authentication && d.blk_device
+            auth_list << ("using " + d.authentication.name + " for " + d.blk_device.name)
+          end
+        end
+        if auth_list.size > 0
+          para(_("Authentication for encrypted devices:")) + list(auth_list)
+        else
+          ""
+        end
```

Comment:
I would improve this method a bit:

```suggestion
        auth_list = @devicegraph.encryptions
          .select(&:supports_authentication?)
          .map { |e| format(_("using %s for %s"), e.authentication.name, e.blk_device.name) }
          
        return "" if auth_list.none?
        
        para(_("Authentication for encrypted devices:")) + list(auth_list)
```

Reply:
Yes, looks better.